widget.go

```diff
@@ -89,7 +89,7 @@ func NewBaseWidget(base string, index uint8, action, actionHold *ActionConfig, b
 }
 
 // NewWidget initializes a widget.
-func NewWidget(base string, kc KeyConfig, bg image.Image) (Widget, error) {
+func NewWidget(dev *streamdeck.Device, base string, kc KeyConfig, bg image.Image) (Widget, error) {
 	bw := NewBaseWidget(base, kc.Index, kc.Action, kc.ActionHold, bg)
```

Comment:
Maybe instead of passing `dev` to every single widget, we could initialize `BaseWidget` with it, and make it accessible that way.